cmd/test.go

```diff
@@ -61,6 +61,11 @@ func init() {
 				Usage:  "The label of the region which the resource will be provision in",
 				EnvVar: "REGION",
 			},
+			cli.StringFlag{
+				Name:   "import-code",
+				Usage:  "The import code to import an existing resource for that resource",
+				EnvVar: "REGION",
```

Comment:
EnvVar should be `IMPORT_CODE`

Reply:
Good catch, I changed it in the latest commit